src/display/api.js

```diff
@@ -1742,7 +1742,8 @@ var WorkerTransport = (function WorkerTransportClosure() {
             var size = width * height;
             var rgbaLength = size * 4;
             var buf = new Uint8Array(size * components);
-            var tmpCanvas = createScratchCanvas(width, height);
+            var canvasFactory = new DOMCanvasFactory();
```

Comment:
It is still will be passed to the CanvasGraphics, but it will be specified at the RenderParameters. And if it is not specified, created at the render().